packages/graphql-codegen/src/testUtils.ts

```diff
@@ -139,3 +147,18 @@ export function newManyToOneField(
     ...opts,
   };
 }
+
+/**
+ * Mimics the contentToString flow from packages/codegen/src/index.ts to test import extension handling.
+ * This reproduces the same logic used in production when saving files.
+ */
+export function renderCodegenFile(file: CodegenFile, config: Config): string {
```

Comment:
I considered trying to reuse the logic in the main codegen, but it was messy. This seems OK for now.